networkx/generators/joint_degree_seq.py

```diff
@@ -0,0 +1,283 @@
+"""Generate graphs with a given joint degree 
+"""
+#    BSD license.
```

Comment:
No - it doesn't need to look like that.  We do not ask the copyright be assigned.  The code should be licensed by the author with BSD or other compatible license in order to be included.